lib/Config/PopulatorData/Templates/NewsletterBlank121Column.php

```diff
@@ -163,7 +163,7 @@ private function getBody() {
                 "blocks" => [
                   [
                     "type" => "text",
-                    "text" => WPFunctions::get()->__("<h2>This template has...</h2>", 'mailpoet'),
+                    "text" => WPFunctions::get()->'<h2>'.__('This template has...', 'mailpoet').'</h2>',
```

Comment:
Hi ramiy, 
thank you for your work, but have you try to execute the code? It is not a valid code.  Also, it doesn't conform to our coding standards. The `.` operator should be separated by a single space on each side.